core/src/main/java/com/yahoo/ycsb/generator/ExponentialGenerator.java

```diff
@@ -62,17 +60,6 @@ public ExponentialGenerator(double percentile, double range)
 
 	/****************************************************************************************/
 	
-	/** 
-	 * Generate the next item. this distribution will be skewed toward lower integers; e.g. 0 will
-	 * be the most popular, 1 the next most popular, etc.
-	 * @param itemcount The number of items in the distribution.
-	 * @return The next item in the sequence.
-	 */
```

Comment:
why was this JavaDoc removed? Couldn't it be merged with nextValue() for this class?